### ISSUE TITLE
[Idea] Module synergy and ability to rotate module

### ISSUE THREAD

**Issue by xdeadstokex** [PavelZinchenko/event-horizon-frontier-builds#319]
e.g: an anti-matter reactor with four nuclear reactors around it will create a synergy, which reduce weight of them all and give 10% power boost, that way, people will choose whether to sacrifice these 4 slot for nuclear reactor or instead use these slots for inertial damper.
The ability to rotate module will be unlocked through the scientist shop. (edit: or just a main feature)

**Comment by Tree3324417** [comment]
synergy makes no sense, is unintuitive, and sound more like an excuse to make dumb things. 
++, no weakness to something, basically asking to scale the meta in some other ways.
complete disapproval to first one in any form or ways

rotate modules is undesired at the very best (mainly modding), but even if some may approve it, being locked behind grinding for some feat is absolutely dumb.

**Comment by Dynamo10101** [comment]
Sorry but this is not cosmoteer 

-calculator 2 on discord

**Comment by xdeadstokex** [comment]
@Tree3324417 
yep, it's literally an excuse to make dumb things, the point is to make the game more diverse and fun to play, not repeative, the in-game module upgrade is nice but after all, it is just pure additive numbers, maybe make some unique upgrade to modules.
In conclusion, that is my dumb idea, just want to make the game more diverse, there are better ways though but I dont know :D
(synergy is not dumb though, it is just that my e.g is dumb, take a look at Backpack hero or Enter the Gungeon, synergy is one of the main features of these games)

**Comment by xdeadstokex** [comment]
@Dynamo10101 thank, I dont know that game, I will try it.
Checked it and it is not what I seek for
